### DISCUSSION TITLE
Notification email

### DISCUSSION THREAD

**Discussion by akasaka99** [ViennaRSS/vienna-rss#1433] [General]
Hello-
I've been using Vienna for 3 months and it's really nice !!!
And I was wondering if there was an plugin or companion app that would allow to get some email notifications when a new  feed is uploaded?
Many thanks !

**Comment by TAKeanice** [comment]
Can you describe your usecase a little more specific? I think Vienna itself actually covers what I assume you want: You can leave it open in the background and you will receive notifications when new articles arrive. You can consider Vienna as "email receiver" for your chosen news feeds.

**Reply by akasaka99** [reply]
Hello-
Yes the app is great on the desktop and i was thinking maybe to get a notification on my email when i'm using my mobile.
Thanks.
